app/src/main/java/com/muzima/service/MuzimaLocationService.java

```diff
@@ -108,7 +109,7 @@ public void onProviderDisabled(String provider) {
                 }
 
                 Log.e(getClass().getSimpleName(), "getLastKnownGPS() == " + location);
-                Toast.makeText(context, "Kindly switch on location in settings.", Toast.LENGTH_LONG).show();
+                Toast.makeText(context, context.getResources().getString(R.string.location_toast_message), Toast.LENGTH_LONG).show();
```

Comment:
Kindly update resource name (location_toast_message) to match convention for naming resources